alembic/versions/bc10fd684e42_remove_work_on_behalf_of_organization_.py

```diff
@@ -0,0 +1,26 @@
+"""remove work_on_behalf_of_organization from Users table
+
+Revision ID: bc10fd684e42
+Revises: bd67c88713b8
```

Comment:
down revision should be ef627578319a since an additional alembic version was added to dev
(locally in your local DB - to make sure DB is aligned I suggest that before pull dev you'll down revise to the version bd67c88713b8)

Reply:
I merged another PR with alembic revision (312c9eb92e40) - FYI - once pr is ready check the latest alembic version and modify the down revision to it.